iohk/NixOps.hs

```diff
@@ -844,6 +855,10 @@ scpFromNode o c (fromNodeName -> node) from to = do
     ExitSuccess -> return ()
     ExitFailure code -> TIO.putStrLn $ "scp from " <> node <> " failed with " <> showT code
 
+triggerCommitMonitor :: Options -> NixopsConfig -> NodeName -> IO ()
+triggerCommitMonitor o c m = do
+  ssh' o c "bash" ["-c", "strings $(pgrep -fal cardano-node-simple | cut -d' ' -f2) 2>/dev/null | egrep '^[0-9a-f]{40,40}$' | xargs logger -t cardano-node-commit-monitor"] m empty
```

Comment:
If we do decide to go forward with this PR, lets change logger to: `systemd-cat -t cardano-node-commit-fetcher -p info`